book/src/pong-tutorial/pong-tutorial-02.md

```diff
@@ -334,7 +334,19 @@ thread 'main' panicked at 'Tried to fetch a resource of type "amethyst::ecs::sto
 Try adding the resource by inserting it manually or using the `setup` method.'
 ```
 
-To turn on the `nightly` feature, run: `cargo +nightly run --features nightly`.
+To turn on the `nightly` feature, enable the `nightly` flag for the Amethyst crate in your Cargo.toml file.
+#### In Dependencies:
+```
+[dependencies]
+amethyst = { version = "X.XX", features = ["nightly"] }
+```
+#### In Separate Table:
```

Comment:
It might be helpful to specify that you should only use one of those 2 options, since both at the same time will cause an error.